AuburnACM/auacm
# John/angular4

This is an upgrade to angular 4 which fixes a few production compilation errors as well.

To test:

1. Clone the repo and checkout this branch
2. Delete the node_modules folder in auacm/angular folder if you have it already
3. Run the setup script if you have not already. 
4. Ensure you have angular/cli version 1.4.0. `ng --version`
5. run `make prod` in the top folder of the repo.
6. cd to `auacm/` and run the project with `./run.py`
7. Test in your browser to make sure the website is running properly.
